Documentation/contributing/development/contributing_guide.rst

```diff
@@ -235,43 +349,86 @@ Pull request review process
    note suitability. Put yourself into the perspective of a future release
    notes reader with lack of context and ensure the title is precise but brief.
 
-   +-----------------------------------+---------------------------------------------------------------------------+
-   | Labels                            | When to set                                                               |
-   +===================================+===========================================================================+
-   | ``dont-merge/needs-release-note`` | Do NOT merge PR, needs a release note                                     |
-   +-----------------------------------+---------------------------------------------------------------------------+
-   | ``release-note/bug``              | This is a non-trivial bugfix                                              |
-   +-----------------------------------+---------------------------------------------------------------------------+
-   | ``release-note/major``            | This is a major feature addition                                          |
-   +-----------------------------------+---------------------------------------------------------------------------+
-   | ``release-note/minor``            | This is a minor feature addition                                          |
-   +-----------------------------------+---------------------------------------------------------------------------+
+   +-----------------------------------+--------------------------------------------------------------------------------------------------------+
+   | Labels                            | When to set                                                                                            |
```

Comment:
This table duplicates the other one further up. What about adding a link to the other one instead as janitors should already be familiar with the release-note labels anyway?